operator/v1/types_dns.go

```diff
@@ -170,6 +184,35 @@ type ForwardPlugin struct {
 	// +optional
 	// +kubebuilder:default:="Random"
 	Policy ForwardingPolicy `json:"policy,omitempty"`
+
+	// transport allows cluster administrators to opt-in to using a DNS-over-TLS
+	// connection between cluster DNS and an upstream resolver(s). Configuring
+	// TLS as the transport at this level without configuring a CABundle will
+	// result in the system certificates being used to verify the serving
+	// certificate of the upstream resolver(s).
+	//
+	// +optional
```

Comment:
tell me what "" means.  I suspect that it's: "Empty value means no opinion and the platform chooses the a default, which is subject to change over time.  Currently the default is blah"